pkg/operator/ceph/csi/util.go

```diff
@@ -99,3 +102,20 @@ func templateToDeployment(name, templatePath string, p templateParam) (*apps.Dep
 	}
 	return &ds, nil
 }
+
+func getPortFromENV(env string, defaultPort uint16) uint16 {
+	port := os.Getenv(env)
+	if strings.TrimSpace(port) == "" {
+		return defaultPort
+	}
+	p, err := strconv.ParseUint(port, 10, 64)
+	if err != nil {
+		logger.Errorf("failed to parse port value for env %s. %+v ,using default port %d", env, err, defaultPort)
```

Comment:
```suggestion
		logger.Debugf("failed to parse port value for env %s. using default port %d. %+v ", env, defaultPort, err)
```

Error object should be at the end, always.